src/main/java/gov/cabinetofice/gapuserservice/web/LoginController.java

```diff
@@ -92,8 +95,27 @@ public RedirectView logout(
         return new RedirectView(colaLogout);
     }
 
+    @GetMapping("/refresh-token")
+    public ResponseEntity<String> refreshToken(@CookieValue(USER_SERVICE_COOKIE_NAME) final String currentToken, final HttpServletResponse response) {
+
+        // presumably the security filter will handle invalid/non-existent tokens and deny access but if not then add code below.
```

Comment:
```suggestion
```

Reply:
Can we leave this in until the security stuff has been worked into the code and we can confirm whether it actually works?